uqcsbot/cowsay.py

```diff
@@ -0,0 +1,141 @@
+
+from typing import Optional, Literal
+
+import discord
+from discord import app_commands
+from discord.ext import commands
+
+from uqcsbot.bot import UQCSBot
+
+# Type Alias for Cow Eyes 
+CowsayMoodType = Literal['Normal', 'Borg', 'Dead', 'Greed', 'Paranoid', 'Stoned', 'Tired', 'Wired', 'Youthful']
+
+class Cowsay(commands.Cog):
+
+    def __init__(self, bot: UQCSBot):
+        self.bot = bot
+        self._max_length = 40
+        self._cow_eyes = dict(
+            Normal = 'oo',
+            Borg = '==',
+            Dead = 'xx',
+            Greed = '$$',
+            Paranoid = '@@',
+            Stoned = '**',
+            Tired = '--',
+            Wired = 'OO',
+            Youthful = '..'
+        )
+
+    @app_commands.command(name="cowsay")
+    @app_commands.describe(
+        message="The message that the cow will say",
```

Comment:
Just making it clear that the messsage can only contain 1000 characters.

```suggestion
        message="The message that the cow will say (Max 1000 chars)",
```